Add GT recipes for Solidified Experience

NotMyWing opened the pull request:
Adds a Fluid Extractor recipe to convert Solidified Experience into 160mb Liquid XP.
Adds a Fluid Solidifier recipe to convert 160mb Liquid XP (ball mold) into Solidified Experience.

Because Experience Solidifiers from ActAdd cannot be automated.

NotMyWing committed:
Add GT recipes for Solidified Experience

ALongStringOfNumbers commented:
Is the fluid extractor recipe really needed if you can get the Liquid XP from DML matter? Or was it just to keep it from ending up as a dead end recipe?

Exaxxion commented:
I think this is an interesting idea. There are alternatives to using solidified XP for most cases, but something like this could be nice.

XP Solidifiers can sort of be automated by dumping liquid XP into an Experience Obelisk with a creative tank until it's full, extracting all the xp as levels (~4000), and then dumping all the levels into a solidifier. This produces a ridiculous amount of solidifed XP that will take a very long time to extract all of from the machine into a proper inventory like a quantum chest for mass crafting.

NotMyWing commented:
It makes sense to have both recipes in place, a dead-end isn't a good thing to have especially when people can mistakenly convert the entirety of their storage into Solidified Experience.

Exaxxion commented:
I think this change looks fine, though I would be interested in your opinions on whether we should change the Solidified XP custom tooltip. I believe it's there because there's no JEI listing for how they're normally made (exp solidifier).

GeoTerra commented:
If you mean that the Experience Solidified isn't in JEI Exa, in dev it is at least. And it has a recipe (an HV Fluid Solidified I believe + 8 Vibrant Plates). Probably wouldn't be a bad idea to add a tooltip to Solidified Experience though.

Exaxxion commented:
What I mean is that I don't think that the experience solidifier comes up as the source of solidified experience if you use JEI recipe lookup on solidified experience. I don't recall there being a tab for it, similar to how there's no tab explaining canola oil comes from the canola press.

Exaxxion committed:
Adjust Solidified Experience tooltip

Better explains how to get Solidified Experience.
Split tooltip across two lines so the box isn't super wide.
